www/js/sectionHeaders.js

```diff
@@ -1,7 +1,9 @@
 var utilities = require("./utilities");
 
+var querySelectorForHeadingsToNativize = 'h1.section_heading, h2.section_heading, h3.section_heading';
+
```

Comment:
Now we can control exactly which level of section headers we are going to overlay with native ones - presently only levels 1 to 3, or h1, h2 and h3.